docs/content/_navigation.json

```diff
@@ -1098,6 +1098,10 @@
           }
         ]
       },
+      {
+        "title": "dlt",
+        "path": "/integrations/dlt"
```

Comment:
Per Pete's suggestion, we may _still_ want to maintain this top level Embedded ELT folder for general messaging, despite the packages being separate.

Reply:
Done in [601b39d](https://github.com/dagster-io/dagster/pull/27071/commits/601b39d6183cc7391a1f6e7237606e4238ce7291)